### PR TITLE
footer

### PR DESCRIPTION
**Pull request by dennisarimi**
Screenshot 2024-01-02 at 11 33 45 PM

### CODE REVIEW

File: src/components/Footer/footer.tsx [hunk 1]
```diff
@@ -2,18 +2,32 @@ import { Fragment, ReactNode, useState, useEffect } from 'react';
...
 import Link from 'next/link';
 import Image from 'next/image';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSquareInstagram, faXTwitter, faLinkedin } from '@fortawesome/free-brands-svg-icons';
+import { faCopyright } from '@fortawesome/free-regular-svg-icons';
 
-interface IProps {}
+interface IProps {
+	socialMedia: {
+		instagram: string;
```
Comment (MatthewOlaka) [inline src/components/Footer/footer.tsx:11]: Could we fix this indent?

**Review by MatthewOlaka — COMMENTED** [review]
Other than the one comment, this looks great but there's a build error coming from vercel... could you investigate why that is?

**Comment by dennisarimi** [comment]
> Other than the one comment, this looks great but there's a build error coming from vercel... could you investigate why that is?

I gave you access to the deployment, maybe try again and see if the build error is still there. There's no error on my side. I added a test comment on the UI which needs to be resolved first to merge, so it shows that "Some checks were not successful" on Git in that regard. Is that the issue?

File: src/components/Footer/footer.tsx (hunk 1 above)
Comment (dennisarimi) [reply src/components/Footer/footer.tsx:11]: Not sure why it's like this, checked github dev and it looks fine, same as on my local. I think its okay

**Comment by MatthewOlaka** [comment]
> > Other than the one comment, this looks great but there's a build error coming from vercel... could you investigate why that is?
> 
> I gave you access to the deployment, maybe try again and see if the build error is still there. There's no error on my side. I added a test comment on the UI which needs to be resolved first to merge, so it shows that "Some checks were not successful" on Git in that regard. Is that the issue?

yeah the some checks were not successful was my worry, but sounds good then!

**Review by MatthewOlaka — APPROVED** [review]
🧚🏼‍♀️

**Pull request merged**